### PR TITLE
Cloud release notes for May 2024

### PR DESCRIPTION
**Pull request by jonat123**
## Description

Release notes for Cloud in May 2024

_What did you add/update/change?_
Added information about new features released in May 2024

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)
ASAP
_When should the content be published?_
ASAP

### CODE REVIEW

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+With the introduction of the Point-In-Time Recovery (PITR) feature on the new "Backups" page, we started a process of being able to backup and restore a whole Umbraco Cloud website.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md:12]:
```suggestion
Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 
```

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# May 2024
+
+## Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md:5]:
```suggestion
* **Upload Custom Database** - You can now upload your Umbraco database to Umbraco Cloud. Add additional text, restoring content is right around the corner.
```

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites.
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md:12]:
```suggestion
With the introduction of the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling back up and restore capabilities for Umbraco Cloud websites.. 
```

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 
+
+This release has enabled you to upload your custom Umbraco database to Umbraco Cloud. Utilizing the current backup functionality or bringing your own database is possible. We expect the possibility of restoring a database to your environment will be available in May or June 2024
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md:14]:
```suggestion
This release allows you to upload your custom Umbraco database to Umbraco Cloud. You can either utilize the current backup functionality or bring your own database. We expect the possibility to restore a database to your environment to be available in May or June 2024.
```

File: umbraco-cloud/release-notes/overview-2024.md [hunk 5]
```diff
@@ -4,6 +4,14 @@ description: Get an overview of release notes for each month in 2024.
 
 # Overview 2024
 
+### [May 2024](2024-05-releasenotes.md)
+
+### Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/overview-2024.md:11]:
```suggestion
* **Upload Custom Database** - You can now upload your Umbraco database to Umbraco Cloud. Add additional text, restoring content is right around the corner.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
The PR looks good, @jonat123 💪 Can you look at the suggestions I've made and let me know if it doesn't make sense?

**Comment by jonat123** [comment]
@eshanrnh Thanks!

Your suggestions looks good to me 😄

I have added them all.

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+## Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database to Umbraco Cloud. Add additional text, restoring content is right around the corner.
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md]:
```suggestion
By introducing the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backups and restore capabilities of Umbraco Cloud websites. 
```

File: umbraco-cloud/release-notes/2024-05-releasenotes.md (hunk 3 above)
Comment (jonat123) [reply umbraco-cloud/release-notes/2024-05-releasenotes.md]: THe sentence will be too long with your suggestion 🙈

I have tried to add another suggestion.

File: umbraco-cloud/release-notes/2024-05-releasenotes.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+## Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database to Umbraco Cloud. Add additional text, restoring content is right around the corner.
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+With the introduction of the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling back up and restore capabilities for Umbraco Cloud websites..
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-05-releasenotes.md:12]:
```suggestion
By introducing the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backups and restore capabilities of Umbraco Cloud websites. 
```

**Pull request merged**